__tests__/pages/curriculum/[lesson].test.js

```diff
@@ -179,7 +179,7 @@ describe('Lesson Page', () => {
     await waitFor(() => expect(container).toMatchSnapshot())
   })
   test('Should render with nulled submissions', async () => {
-    query['lesson'] = 2
+    query['lesson'] = 'js1'
```

Comment:
Do we really need to have `js0` and `js1` for tests? It seems a bit unnecessary.

Reply:
The current way the pages are setup it fetches all the lessons on initial load and then determines which lesson to render based on the the query parameter. So we do have to pass in the slug into our mocked next/router.

once this change goes through I would like to refactor both the review page and lesson page to be static generated and  then at that point we could possibly no longer need to pass the slug in